templates/Search/results.html.twig

```diff
@@ -1,49 +1,53 @@
+        </h6>
+    </div>
 
-  <ul class="media-list">
-    {% for coaster in coasters %}
-      <li class="media panel-body stack-media-on-mobile">
-        <div class="media-left">
-          {% if coaster.item.mainImage is not null %}
-            <img src="{{ pictures_cdn }}/96x96/{{ coaster.item.mainImage.filename }}"
-                 class="img-rounded"
-                 style="width:96px;"
-                 alt="{{ coaster.item.name }}">
-          {% endif %}
-        </div>
+    <ul class="media-list">
+        {% for coaster in coasters %}
+            <li class="media panel-body stack-media-on-mobile">
+                <div class="media-left">
+                    {% if coaster.item.mainImage is not null %}
+                        <img src="{{ pictures_cdn }}/96x96/{{ coaster.item.mainImage.filename }}"
+                             class="img-rounded"
+                             style="width:96px;"
+                             alt="{{ coaster.item.name }}">
+                    {% endif %}
+                </div>
 
-        <div class="media-body">
-          <h6 class="media-heading text-semibold">
-            <a href="{{ path('bdd_show_coaster', {'slug': coaster.item.slug}) }}">
-              {{ coaster.item.name }}
-            </a>
-            -
-            <a href="{{ path('park_show', {'slug': coaster.item.park.slug}) }}">
-              {{ coaster.item.park.name }}
-            </a>
-          </h6>
+                <div class="media-body">
+                    <h6 class="media-heading text-semibold">
+                        <a href="{{ path('bdd_show_coaster', {'slug': coaster.item.slug}) }}">
+                            {{ coaster.item.name }}
+                        </a>
+                        -
+                        <a href="{{ path('park_show', {'slug': coaster.item.park.slug}) }}">
+                            {{ coaster.item.park.name }}
+                        </a>
+                    </h6>
 
-          <ul class="list-inline list-inline-separate text-muted mb-10">
-            {% if coaster.item.materialType %}
-              <li>{{ coaster.item.materialType }}</li>
-            {% endif %}
-            {% if coaster.distance is defined %}
-              <li>{{ 'search_index.km_away'|trans({'%num%': coaster.distance|number_format(0)}) }}</li>
-            {% endif %}
-          </ul>
-        </div>
-      </li>
-    {% endfor %}
-  </ul>
+                    <ul class="list-inline list-inline-separate text-muted mb-10">
+                        {% if coaster.item.materialType %}
+                            <li>{{ coaster.item.materialType }}</li>
+                        {% endif %}
+                        {% if coaster.distance is defined %}
+                            <li>
+                                {% set distance = units.km_or_mi(isImperial, coaster.distance) %}
+                                {{ 'search_index.km_away'|trans({'%num%': distance.value, '%unity%': distance.unit }) }}
```

Comment:
It could be useful for another feature to have the value and the unit separated, couldn't it? I can revert to my previous version of the functions if you want.

Is `number_format(0)` useful if we already have `round()` in the function?

Reply:
Not a fan of setting a variable first every time we need it
Plus it's not required at the moment
Let me know what you think but I would revert

Reply:
Done!